/**
 * This file is part of the "libfnord" project
 *   Copyright (c) 2015 Paul Asmuth
 *
 * FnordMetric is free software: you can redistribute it and/or modify it under
 * the terms of the GNU General Public License v3.0. You should have received a
 * copy of the GNU General Public License along with this program. If not, see
 * <http://www.gnu.org/licenses/>.
 */
package cm;

message PartitionInfo {

  required string partition_key = 1;

  optional string table_name = 2;

  optional bool exists = 5;

  optional string checksum = 6;

  optional uint64 cstable_version = 7;

};
